tests/cucumber/features/authentication/authentication.feature

```diff
@@ -7,3 +6,13 @@ Feature: Login
   Background:
     Given I am on the home page
     Given I am signed out
+
+  @dev
+  Scenario: Create account
+    Given "test@test.com" has not registered
```

Comment:
Maybe I should just remove this. I am not sure how to altar the Users collections without using mongo. Do you think it would be wise to make a server method to remove a user or no?